### PR TITLE
APlaylistEntryMessage base test

### PR DESCRIPTION
**Pull request by theJenix**
Pulled generic class APlaylistEntryMessage tests in a test base class, and changed the current playlist messages to use that class.

Commented out and failed PlayStatusMessageTest...this needs to be reimplemented

### CODE REVIEW

**Comment by reidbaker** [comment]
Might want to pull davids request in first. His PlayStatusMessageTest can be found here. https://github.com/dgreenhalgh/fanClub/commit/36e9b05ca2b03e5bbe3ba0e8dca765886a87e5a7

**Comment by theJenix** [comment]
His changes have been m:ear:ged

**Comment by reidbaker** [comment]
you merged PlayStatusMessageTest with the inverse of what I would have done

File: SoundStream-test/src/com/lastcrusade/soundstream/net/message/AllMessageTests.java [hunk 1]
```diff
@@ -24,12 +24,15 @@
 import org.junit.runners.Suite.SuiteClasses;
 
 @RunWith(Suite.class)
-@SuiteClasses({ AddToPlaylistMessageTest.class, BumpSongOnPlaylistMessageTest.class,
-        ConnectGuestsMessageTest.class, FindNewGuestsMessageTest.class,
-        FoundGuestsMessageTest.class, LibraryMessageTest.class,
-        MessengerTest.class, PauseMessageTest.class, PlayMessageTest.class,
-        RemoveFromPlaylistMessageTest.class, RequestSongMessageTest.class,
-        SkipMessageTest.class, UserListMessageTest.class, PlaylistMessageTest.class })
+@SuiteClasses({ AddToPlaylistMessageTest.class,
```
Comment (reidbaker) [inline SoundStream-test/src/com/lastcrusade/soundstream/net/message/AllMessageTests.java:27]: Weird thing to ask but could you put all of these on their own line? That way we can see what was added or removed easier.

File: SoundStream-test/src/com/lastcrusade/soundstream/net/message/PlayStatusMessageTest.java [hunk 2]
```diff
@@ -1,30 +1,36 @@
-/*
```
Comment (reidbaker) [inline SoundStream-test/src/com/lastcrusade/soundstream/net/message/PlayStatusMessageTest.java:1]: This should have our copyright statement

**Comment by reidbaker** [comment]
All the new files should have the copy right info.

File: SoundStream-test/src/com/lastcrusade/soundstream/net/message/AllMessageTests.java (hunk 1 above)
Comment (theJenix) [inline SoundStream-test/src/com/lastcrusade/soundstream/net/message/AllMessageTests.java:27]: This is actually generated code.  Do we have a document describing how to write tests?  If so, we should add a step to regenerate the suite.

File: SoundStream-test/src/com/lastcrusade/soundstream/net/message/PlayStatusMessageTest.java (hunk 2 above)
Comment (theJenix) [inline SoundStream-test/src/com/lastcrusade/soundstream/net/message/PlayStatusMessageTest.java:1]: Do we have a script for that?

Comment (reidbaker) [inline SoundStream-test/src/com/lastcrusade/soundstream/net/message/PlayStatusMessageTest.java:1]: Yes it was part of greeny's pull request. But I am not sure if it checks to
see of the copyright stuff is already there

Comment (dgreenhalgh) [inline SoundStream-test/src/com/lastcrusade/soundstream/net/message/PlayStatusMessageTest.java:1]: Yes it does :sunglasses:

**Comment by theJenix** [comment]
Dun dun dun duuuuuuuuun.

**Comment by reidbaker** [comment]
I think this is ready to be pulled in. :put_litter_in_its_place:

**Pull request merged**